codecov.yml

```
@@ -0,0 +1,9 @@
+coverage:
+  range: "60...95"
+  status:
+    project:
+      default:
+        threshold: 1%
+
+comment:
+  layout: 'diff'
```

```suggestion
  layout: 'diff'

github_checks: false
```

Let's stop letting codecov fail CI.